api/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM node:lts-alpine
+FROM node:lts-buster
 ENV NODE_ENV=production
 WORKDIR /usr/src/app
 COPY ["package.json", "package-lock.json*", "npm-shrinkwrap.json*", "./"]
-RUN npm install --production --silent && mv node_modules ../
+RUN npm instal --silent && mv node_modules ../
```

Comment:
`instal`?

Reply:
Lol, instal surprising works but I'll change it XD